LocalSettings.php

```diff
@@ -2475,6 +2475,15 @@
 	'wgAllowHTMLEmail' => [
 		'default' => true,
 	],
+	'wgEnableSpecialMute' => [
+		'default' => true,
+	],
+	'wgEnableUserEmailMuteList' => [
+		'default' => true,
+	],
+	'wgEchoPerUserBlacklist' => [
```

Comment:
This should be moved to the Echo section, at line 1230.

Reply:
Apologies, done